modules/gdnative/gdnative/callable.cpp

```diff
@@ -30,223 +30,24 @@
 
 #include "gdnative/callable.h"
 
-#include "core/io/resource.h"
 #include "core/variant/callable.h"
 #include "core/variant/variant.h"
```

Comment:
Variant might be unused now, no?

Reply:
The problem is that Callable needs Variant (it's only forward-declared in the header), so I need a full definition otherwise it can't compile.